selfdrive/car/volkswagen/carcontroller.py

```diff
@@ -93,9 +93,8 @@ def update(self, c, CS, frame, ext_bus, actuators, visual_alert, left_lane_visib
           # Cancel ACC if it's engaged with OP disengaged.
           self.graButtonStatesToSend = BUTTON_STATES.copy()
           self.graButtonStatesToSend["cancel"] = True
-        elif c.enabled and CS.esp_hold_confirmation:
-          # Blip the Resume button if we're engaged at standstill.
-          # FIXME: This is a naive implementation, improve with visiond or radar input.
+        elif c.cruiseControl.override:
```

Comment:
```suggestion
        elif c.cruiseControl.override and CS.esp_hold_confirmation:
```

I'd like to retain the extra stock-ACC-standstill verification. See #22176 for history.

Reply:
On further review, I was not aware of `cruiseState.standstill` back when I started reading the ABS/ESP hold confirmation signal. Probably better to populate that so the check is done in higher level controlsd.